Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Eye Changes for Ethnicities

## Changelog:
- Gave **Draenei** `magic_no_pupils_eyes` and changed the brightness to 0.5.
- ~~Gave **Frostborn Dwarves** `magic_no_pupils_eyes`~~ and changed the brightness to 0.5.
  - Reverted back to `magic_eyes` in 997e635ab0489d202845889b9a3f9c706de1039a.
- Gave **Gilblin** and **Ice Trolls** `glowing_iris` and changed brightness to 1.
- Gave **Frost Vrykul**, **Kvaldir** and **Drust** `magic_no_pupils_eyes` and changed their brightness to 0.5.
- Readjusted elf eye brightness to 0.5, down from 1.
- Readjusted troll eye brightness to 0.5, up from 0.35.
- Readjusted **Frost Vrykul** and **Drust** eye color ranges so they include slightly lighter variations.

<details><summary>Images</summary>

![image](https://user-images.githubusercontent.com/78575425/129946859-f19c4b3b-4d0a-4abd-8daf-0ffb3c89614a.png)
![image](https://user-images.githubusercontent.com/78575425/129946905-4b85af6a-5984-4f11-a1c5-c8d52ad8969b.png)
![image](https://user-images.githubusercontent.com/78575425/129947071-65bbf8e2-0f08-4127-9a84-41fed64a5dca.png)
![image](https://user-images.githubusercontent.com/78575425/129947023-549f181f-2d37-4552-be10-799c9324c48c.png)
![image](https://user-images.githubusercontent.com/78575425/129947263-1ba4e7bc-2fb6-4c39-a523-2fdb6c01c302.png)
![image](https://user-images.githubusercontent.com/78575425/129947298-6deec33f-4569-4eda-b49b-e2564ea315cd.png)
![image](https://user-images.githubusercontent.com/78575425/129947364-8389b451-01e3-4ef9-8924-268d8421ebde.png)

</details>

### Implemented Suggestions
- Gave **Ice Trolls** more eye color variation in e96b155526bca4f9366f9a99cb0285b3fceb08e2 (suggested by @Remerod).
  - They may now generate with blue, cyan, purple, and pink eyes in addition to red and orange eyes. Red and orange eyes are, by far, the most common variation.
- Changed **Ice Troll** eyes to glowing_iris in 04f0e888262418b9761b83b56f0147bd712bf86f (suggested by @Remerod).
- Changed **Gilblin** eyes to glowing_iris 04f0e888262418b9761b83b56f0147bd712bf86f (suggested by @Remerod).


# How to test:
- View the changed ethnicities either using the Ruler Designer, [Debug Portrait Editor](https://ck3.paradoxwikis.com/Console_commands#Enabling_debug_mode), or by looking for their cultures in-game with the Character Finder (Draenei and Gilblin cannot be found using the last method).
- Provide feedback on the new eyes and if you think any other ethnicities should have their eyes changed or altered in some way (colors, brightness, eye type, etc.). You may use [this album](https://imgur.com/a/pQKUdkd) to make comparisons.
